### ISSUE TITLE
Got error in my bot.

### ISSUE THREAD

**Issue by anshtyagi0** [Deivu/Shoukaku#100]
```
0|java  | 2022-08-27 11:43:11.867  INFO 26406 --- [  XNIO-1 task-8] l.server.player.AudioLoaderRestHandler   : Got request to load for identifier "ytmsearch:295"
0|java  | 2022-08-27 11:43:12.260  INFO 26406 --- [ader-2-thread-1] lavalink.server.player.AudioLoader       : Loaded playlist Search music results for: 295
1|index | SyntaxError: Unexpected token  in JSON at position 0
1|index |     at JSON.parse (<anonymous>)
1|index |     at ShoukakuRest.fetch (/home/ansh/Cosmic/node_modules/shoukaku/src/node/ShoukakuRest.js:112:21)
1|index |     at processTicksAndRejections (node:internal/process/task_queues:96:5)
1|index |     at async ShoukakuRest.resolve (/home/ansh/Cosmic/node_modules/shoukaku/src/node/ShoukakuRest.js:50:22)
1|index |     at async kazagumoSearch.search (/home/ansh/Cosmic/node_modules/kazagumo/src/kazagumoSearch.js:48:24)
1|index |     at async kazagumoPlayer.search (/home/ansh/Cosmic/node_modules/kazagumo/src/kazagumoPlayer.js:161:16)
1|index |     at async playerhandler (/home/ansh/Cosmic/src/utils/functions.js:84:20)
1|index |     at async Object.run (/home/ansh/Cosmic/src/events/Client/setupSystem.js:42:9)
1|index | SyntaxError: Unexpected token  in JSON at position 0
1|index |     at JSON.parse (<anonymous>)
1|index |     at ShoukakuRest.fetch (/home/ansh/Cosmic/node_modules/shoukaku/src/node/ShoukakuRest.js:112:21)
1|index |     at processTicksAndRejections (node:internal/process/task_queues:96:5)
1|index |     at async ShoukakuRest.resolve (/home/ansh/Cosmic/node_modules/shoukaku/src/node/ShoukakuRest.js:50:22)
1|index |     at async kazagumoSearch.search (/home/ansh/Cosmic/node_modules/kazagumo/src/kazagumoSearch.js:48:24)
1|index |     at async kazagumoPlayer.search (/home/ansh/Cosmic/node_modules/kazagumo/src/kazagumoPlayer.js:161:16)
1|index |     at async playerhandler (/home/ansh/Cosmic/src/utils/functions.js:84:20)
1|index |     at async Object.run (/home/ansh/Cosmic/src/events/Client/setupSystem.js:42:9) Promise {
1|index |   <rejected> SyntaxError: Unexpected token  in JSON at position 0
1|index |       at JSON.parse (<anonymous>)
1|index |       at ShoukakuRest.fetch (/home/ansh/Cosmic/node_modules/shoukaku/src/node/ShoukakuRest.js:112:21)
1|index |       at processTicksAndRejections (node:internal/process/task_queues:96:5)
1|index |       at async ShoukakuRest.resolve (/home/ansh/Cosmic/node_modules/shoukaku/src/node/ShoukakuRest.js:50:22)
1|index |       at async kazagumoSearch.search (/home/ansh/Cosmic/node_modules/kazagumo/src/kazagumoSearch.js:48:24)
1|index |       at async kazagumoPlayer.search (/home/ansh/Cosmic/node_modules/kazagumo/src/kazagumoPlayer.js:161:16)
1|index |       at async playerhandler (/home/ansh/Cosmic/src/utils/functions.js:84:20)
1|index |       at async Object.run (/home/ansh/Cosmic/src/events/Client/setupSystem.js:42:9)
1|index | }
```

**Comment by Deivu** [comment]
Node Version?
Shokaku Version?

**Comment by anshtyagi0** [comment]
node - v16.17.0
shokaku - 8.15.0

**Comment by Deivu** [comment]
Fixed in 3.2.0
